samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -317,6 +323,7 @@ This sample uses the following |NCS| libraries:
 * :ref:`bt_mesh_light_ctrl_srv_readme`
 * :ref:`bt_mesh_scene_srv_readme`
 * :ref:`bt_mesh_sensor_srv_readme`
+* :ref:`bt_mesh_large_comp_data_srv_readme`
```

Comment:
```suggestion
* :ref:`zephyr:bluetooth_mesh_lcd_srv`
```
The Large Composition Data server is part of Mesh core functionality in upstream.

Reply:
that explains it! thanks :)